juniper_official/routing/collect-isis-lsdb-neighbsysid-v1.rule

```diff
@@ -0,0 +1,99 @@
+/*
+ * Collects ISIS lsdb data till neighbor-sysid for metric values from releases between 20.2R1 and 22.2R3
+ * REQ: IGP RCA- https://paragon-automation.atlassian.net/browse/REQ-1349
+ */
+ healthbot {
+    topic routing.isis {
+        rule collect-isis-lsdb-neighbsysid-v1 {
```

Comment:
What is v1 and v2 in the rule names?

Reply:
I have used v1,v2 for the following rules: collect-isis-lsp-retranx, collect-isis-lsdb-neighbsysid as these (v1,v2)files were retrieving the same set of fields but varies with number of keys,

so here,
collect-isis-lsp-retranx-v1 -->works for rel <23.2R2 | keys [ instance-name interface-id ]
collect-isis-lsp-retranx-v2 -->works for rel >=23.2R2 | keys [ instance-name interface-id level-number ]

collect-isis-lsdb-neighbsysid-v1 -->works for releases between 20.2R1 and 22.2R3 | keys [ instance-name level-number lsp-id tlv-type neighbor-system-id ]
collect-isis-lsdb-neighbsysid-v2 -->works for releases >=22.3R1 | keys [ instance-name level-number lsp-id tlv-type neighbor-system-id instance-id ]

Reply:
Please add this details as comments in the rules

Reply:
Also can you confirm that the rules are being instantiated based on the SW version?

Reply:
yes, it will be instantiated based on version